alias/dlgalias.c

```diff
@@ -191,15 +190,15 @@ static void alias_menu(char *buf, size_t buflen, struct AliasMenuData *mdata)
 
   int t = -1;
   bool done = false;
-  char helpstr[1024];
 
   struct Menu *menu = mutt_menu_new(MENU_ALIAS);
   struct MuttWindow *dlg = dialog_create_simple_index(menu, WT_DLG_ALIAS);
+  dlg->help_data = AliasHelp;
+  dlg->help_menu = MENU_ALIAS;
 
   menu->make_entry = alias_make_entry;
   menu->tag = alias_tag;
   menu->title = _("Aliases");
-  menu->help = mutt_compile_help(helpstr, sizeof(helpstr), MENU_ALIAS, AliasHelp);
```

Comment:
I think you can get rid of the `help` member of `struct Menu` altogether :)

Reply:
D'oh! How'd I forget that?
done